src/PrecompilationWebSite/PrecompilationWebSite.kproj

```diff
@@ -1,17 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
```

Comment:
correct.  the directory is otherwise empty.  content was moved to test\WebSites\PrecompilationWebSite.